scripts/rev_cron.php

```diff
@@ -0,0 +1,197 @@
+<?php
+if (file_exists(dirname(__FILE__).'/../config.inc.php')) {
+    require_once dirname(__FILE__).'/../config.inc.php';
+} else {
+    require_once dirname(__FILE__).'/../config.sample.php';
+}
+
+//Establish mediahub connection
+$media_hub = new UNL_MediaHub();
+
+$db = Doctrine_Manager::getInstance()->getCurrentConnection();
+
+//Get all orders that have not been completed.
+$orders = new UNL_MediaHub_RevOrderList(array('all_not_complete'=>true));
+
+/**
+ * Order status life cycle:
+ * 1. STATUS_MEDIAHUB_CREATED (order was created by mediahub, not yet created in rev)
+ * 2. created (created in rev)
+ * ...
+ * 3. complete (finished in rev, files ready)
+ * 4. STATUS_MEDIAHUB_FINISHED
+ */
+
+$rev = UNL_MediaHub_RevAPI::getRevClient();
+
+if (!$rev) {
+    throw new Exception('Unable to get the Rev client', 500);
+}
+
+$has_output = false;
+
+//Loop through them and check with Rev.com to see their status.
+foreach ($orders->items as $order) {
+    echo $order->id . ': ' . $order->status . PHP_EOL;
+    echo "\tstatus: " . $order->status . PHP_EOL;
+    
+    $media = $order->getMedia();
+    
+    switch ($order->status) {
+        case UNL_MediaHub_RevOrder::STATUS_MEDIAHUB_COMPLETE:
+        case UNL_MediaHub_RevOrder::STATUS_ERROR:
+            //Nothing to do here (in fact, this code should never execute)
```

Comment:
Should you throw an Exception if you truely believe this should be a dead block?